### PR TITLE
[MIG][11.0] website_seo_redirection

### PR DESCRIPTION
**Pull request by jcarlosmontoya**
Migrate website_seo_redirection module 10.0 to 11.0

### CODE REVIEW

**Comment by simahawk** [comment]
@jcarlosmontoya I'm not sure we need this on v11: redirection tool is already there. It's hidden: you have to enable debug mode to see it in website config. Any improvements we can cherry pick from here?

**Comment by jcarlosmontoya** [comment]
thank you @simahawk , but i'm using for blog, y does not work, 
url_from: /blog/blog1/myblogexample
url_to: /myblogexample

Am i doing somethig wrong?

**Comment by simahawk** [comment]
@jcarlosmontoya seems so. Can you reproduce it on runbot and file and issue  at https://github.com/odoo/odoo/issues ?

**Comment by tarteo** [comment]
It's better to use standard model website.redirect and only use this module to render the Atom feed (website_blog) and sitemap.xml correctly. The standard functionality of Odoo works fine except for that.

**Comment by simahawk** [comment]
@jcarlosmontoya ping :)

File: website_seo_redirection/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
```
Comment (nikul-serpentcs) [inline website_seo_redirection/__init__.py:2]: @jcarlosmontoya Remove Copyright from all `__init__.py` file

File: website_seo_redirection/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# © 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# © 2018 Juan Carlos Montoya <jcarlosmontoyach@gmail.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+{
+    "name": "Website SEO Redirection",
+    "summary": "Redirect any controller to the URL of your dreams",
+    "version": "11.0.1.0.0",
+    "category": "Website",
+    "website": "https://tecnativa.com/",
```
Comment (nikul-serpentcs) [inline website_seo_redirection/__manifest__.py:10]: @jcarlosmontoya Change URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)

**Review by nikul-serpentcs — COMMENTED** [review]
Small Change

**Comment by pedrobaeza** [comment]
Is this still needed with the new Odoo core features? I have migrated without this module without losing the features my customers were using.

**Comment by rafaelbn** [comment]
Hi @jcarlosmontoya , hi all !

As you have commented this is already in the core



So IMHO opinion we shouldn't migrate this module.

@Yajo as original contributor what do you think?

Thanks

**Comment by yajo** [comment]
As other have explained above, this addon is no more needed in v11.

Besides, apart of the advanced page that @rafaelbn showed above, there is a nice UI for setting redirections as seen here:

captura de pantalla de 2018-10-08 09-22-15

**Comment by eLBati** [comment]
@simahawk @tarteo @Yajo @rafaelbn 
I think odoo core does not cover every use case covered by this module.
For example:

> #. Go to *Website Admin > SEO Redirections > Create*.
#. Set ``/blog/our-news-1/post/amazing-post-23`` as *Original URL*.
#. Set ``/amazing`` as *Destination URL*.
#. Enable *Relocate controller*.

> Now navigate to any of both URLs, and you will get to the blog post, but with
the URL you wanted.

Am I wrong?

**Comment by yajo** [comment]
Possibly there's some case that's not covered, but it should be probably analyzed if it's really needed, and reimplemented as an extension to the new upstream system in case it is.

**Pull request closed without merging**